CMakeLists.txt

```diff
@@ -2,6 +2,13 @@
 # Created by Robin Rowe 2020-05-01
 # License: License Public Domain
 
+# The process of building pForth involves several steps. 
+# cmake .. 
+# cd fth
+# ../build/Win32/Debug/pforth -i system.fth
```

Comment:
Can cmake do this?

Reply:
# mkdir -p build/x64
# cd build/x64
# cmake ../.. -A x64
# msbuild Pforth.sln

For 32-bit build, instead of x64 say Win32. 

There is an alternative to msbuild that does about the same thing but isn't as verbose:

# devenv /build Debug Pforth.sln

To make cmake do a post-build step would use add_custom_command() or add_custom_target depending upon the intention.

It is possible to use cmake with the --build switch to call the platform's build too implicitly. I haven't tried it. Is something like this:

# cmake --build . --target ALL_BUILD --config Release -- /nologo /verbosity:minimal /maxcpucount 

By the way, because cmake is normally used to build out of source tree, it will not collide if you leave your legacy make file in your repo for folks who may be using that.